airbyte-lib/airbyte_lib/_processors.py

```diff
@@ -45,6 +45,7 @@
 
 
 DEFAULT_BATCH_SIZE = 10_000
+DEBUG_MODE = False  # Set to True to enable additional debug logging.
```

Comment:
Nothing we need to change here, but later we might consider optionally having a single DEBUG_MODE flag, perhaps in a shared module like `airbyte_lib._constants` or `airbyte_lib.constants`.